test/integration/ProjectBrowserTest.kt

```diff
@@ -37,10 +43,10 @@ class ProjectBrowserTest: IntegrationTest() {
             cloneAndOpenProject(composeRule, source = SAMPLE_DATA_PATH, destination = testID)
 
             StudioState.project.current!!.directory.asDirectory().tryCreateDirectory(createdDirectoryName)
-            wait(composeRule, 500)
+            waitAndRecompose(composeRule, Delays.RECOMPOSE)
```

Comment:
Now let's take advantage of the default param by using it everywhere applicable
```suggestion
            waitAndRecompose(composeRule)
```

Reply:
Done.